package.json

```diff
@@ -144,12 +148,14 @@
     "sass-resources-loader": "^1.3.0",
     "source-map-loader": "^0.2.1",
     "ts-jest": "^22.4.1",
+    "ts-node": "^5.0.1",
```

Comment:
why is this now a devDependency? its already a regular dependency

Reply:
does it need to be a regular dependency? its should not be bundled with the app.

Reply:
we use it to run scripts on Heroku

Reply:
doesn't it `npm install` it and runs the command inside the dir?